base/components/PropControls/PropControlUpload.jsx

```diff
@@ -11,22 +11,34 @@ import Icon from '../Icon';
 import LinkOut from '../LinkOut';
 import { luminanceFromHex } from '../Colour';
 
+
 /** MIME type sets */
 const imgTypes = 'image/jpeg, image/png, image/svg+xml';
 const videoTypes = 'video/mp4, video/ogg, video/x-msvideo, video/x-ms-wmv, video/quicktime, video/ms-asf';
+const fontTypes = 'font/ttf, font/otf, font/woff, font/woff2';
+
+
+/** Uploader types which take a 100x100 square thumbnail */
+const typesWithThumbnail = { imgUpload: true, videoUpload: true, bothUpload: true };
+/** Style block for a 100x100 square thumbnail */
+const thumbnailStyle = { width: '100px', height: '100px', position: 'relative' };
```

Comment:
Wouldn't this be better done in css, where it can be re-styled?

Reply:
As you say - not a regression but can be done better. I'll make a note to clean it up post-release.